### PR TITLE
[gardening] Fix typos. Fix word processor artefacts. Fix duplicate includes. Fix clang-tidy warnings.

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

File: lib/AST/ArchetypeBuilder.cpp [hunk 1]
```diff
@@ -2063,7 +2063,7 @@ void ArchetypeBuilder::visitPotentialArchetypes(F f) {
 
 namespace {
   using PotentialArchetype = ArchetypeBuilder::PotentialArchetype;
-}
+} // namespace
```
Comment (jrose-apple) [inline lib/AST/ArchetypeBuilder.cpp]: "end anonymous namespace"?

File: stdlib/public/SDK/os/os_log.m [hunk 2]
```diff
@@ -71,7 +71,7 @@
 
 typedef struct {
     os_log_fmt_cmd_flags_t cmd_flags : 4;
-    os_log_fmt_cmd_type_t  cmd_type : 4;
+    os_log_fmt_cmd_type_t cmd_type : 4;
```
Comment (jrose-apple) [inline stdlib/public/SDK/os/os_log.m]: These files are already not formatted Swift-ly; I don't think you should bother editing them for formatting.

**Comment by practicalswift** [comment]
@jrose-apple Good points! Updated and pushed. Thanks for reviewing!

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

**Comment by practicalswift** [comment]
@swift-ci please test and merge

**Pull request merged**